docker/docker-compose.yml

```diff
@@ -6,9 +6,9 @@ services:
     volumes:
       - esdata:/usr/share/elasticsearch/data
       - ./elasticsearch/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
-    #ports:
-    #  - "9200:9200"
-    #  - "9300:9300"
+    ports:
+      - "9200:9200"
```

Comment:
@fxprunayre It does not matter that you publish ports 9200 and 9300 on localhost. the dashboard container will still don't find es on localhost. However, it will find ES on: elasticsearch:9200 and elasticsearch:9300, regardless we publish them.